scantool/scantool_850.c

```diff
@@ -683,6 +683,14 @@ static enum cli_retval cmd_850_ping(int argc, UNUSED(char **argv)) {
  * scaled value.
  */
 static void interpret_value(enum l7_namespace ns, uint16_t addr, UNUSED(int len), uint8_t *buf) {
+#if 0
```

Comment:
please add the same short state you have in the commit message about why this block is `if #0`

Reply:
Done.